circle.yml

```diff
@@ -19,9 +19,17 @@ dependencies:
       source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh &&
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
-      conda install pytest &&
+      conda install pytest entrypoints &&
       conda install pytest-cookies -c conda-forge &&
-      conda install ciocheck -c conda-forge;
+      conda install versioneer -c conda-forge &&
+      conda install ciocheck -c conda-forge &&
+      pip install -U pip;
```

Comment:
errors with entrypoints

`error: Could not find suitable distribution for Requirement.parse('entrypoints>=0.2.2')`